docs/tables/aws_amplify_app.md

```diff
@@ -0,0 +1,130 @@
+# Table: aws_amplify_app
+
+Represents the different branches of a repository for building, deploying, and hosting an Amplify app.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  app_id,
```

Comment:
For basic info, could you please keep 4-6 columns like name, arn, app_id, platform, and create_time (only primary ones)

Reply:
I like to provide all the names so that the end user knows what fields are present in the table. 

Granted they could go for `.inspect table_name` but that would require that the user runs the CLI.